docs/zkapps/tutorials/10-account-updates.mdx

```diff
@@ -0,0 +1,381 @@
+type Legend = { [pk: string]: string };
+
+```
+
+We'll build our legend as follows:
+
+```typescript
+  const legend = {
+    [proofsOnlyAddr.toBase58()]: 'proofsOnlyZkApp',
+    [secondaryAddr.toBase58()]: 'secondaryZkApp',
+    [deployerAccount.toPublicKey().toBase58()]: 'deployer',
+  };
+```
+
+Then we'll deploy our smart contracts as follows, and visualize the transaction:
+
+```typescript
+  const deploy_txn = await Mina.transaction(deployerAccount, () => {
+    AccountUpdate.fundNewAccount(deployerAccount, 2);
+    proofsOnlyInstance.deploy({ zkappKey: proofsOnlySk });
+    secondaryInstance.deploy({ zkappKey: secondarySk });
+  });
+
+  await deploy_txn.prove();
+  deploy_txn.sign([ proofsOnlySk, secondarySk ]);
+
+  await showTxn(deploy_txn, 'deploy_txn', legend);
+
+  await deploy_txn.send();
+```
+
+This yields the following visualization of `deploy_txn`. 
+
+![](https://raw.githubusercontent.com/es92/zkApp-examples/main/10-account-updates/deploy_txn.png)
+
+This is best opened in a new tab to read the contents ([link](https://raw.githubusercontent.com/es92/zkApp-examples/main/10-account-updates/deploy_txn.png)).
+
+The deploy transaction includes 5 accountUpdates. In order from left to right;
+
+1. This accountUpdate takes the new account fee from the deployer for deploying the zkApps. Note the `-2` on the `balanceChange` field
+2. This update deploys the `proofsOnlyZkApp` instance. Note the permissions, all set to the values in the zkApp's deploy field, and the `preconditions`, asserting its a new account.
+3. This update initializes the `proofsOnlyZkApp`. Note the precondition that it can't already be in a proved state.
+4. This update deploys an instance of `secondaryZkApp`. Note the permissions here set to default compared to the deployment in the `proofsOnlyZkApp` example.
+5. This update initializes the `secondaryZkApp` instance.
+
+When the transaction is run on-chain, all of these accountUpdates will be checked, and applied. Note that there is no actual "execution" of each AccountUpdate provided - instead each AccountUpdate includes an associated signature or proof, corresponding to the verification key in the zkApp.
+
+Next, we'll call `add` on our instance of `proofsOnlyZkApp`:
+
+```typescript
+  const txn1 = await Mina.transaction(deployerAccount, () => {
+    proofsOnlyInstance.add(Field(4));
+  });
+
+  await txn1.prove();
+
+  await showTxn(txn1, 'txn1', legend);
+
+  await txn1.send();
+```
+
+This yields the following visualization of `txn1`
+
+![](https://raw.githubusercontent.com/es92/zkApp-examples/main/10-account-updates/txn1-copy.png)
+
+See download link [here](https://raw.githubusercontent.com/es92/zkApp-examples/main/10-account-updates/txn1-copy.png).
+
+Now we have two AccountUpdates, where one is a child of the other. The parent corresponds to the `add` method call - and the child corresponds to the `this.incrementCalls()` call that the parent makes.
```

Comment:
I think the tutorial is missing information, or at least a hint, about what it means that "one update is the child of another". I'd recommend to add a sentence about that here. Or maybe you want to save that for the next tutorial?

You could also just link to this section: https://docs.minaprotocol.com/zkapps/how-to-write-a-zkapp#payments-and-more-on-public-inputs
There, the meaning of account update as public inputs is explained in some detail

Reply:
Like, you could say that here, one update is the child of another because it was called by the other one, and that this implies that the parent has the child included as part of its proof.